src/providers/FusionAuthProvider.tsx

```diff
@@ -32,7 +34,7 @@ export const FusionAuthProvider: React.FC<Props> = ({
         (functionType: FunctionType, queryParams: Record<string, string>) => {
             const query = new URLSearchParams(queryParams);
 
-            return `${baseUrl}/${functionType}?${query}`;
+            return `${baseUrl}/oauth2/${functionType}?${query}`;
```

Comment:
Just a comment, this is fine for now, but we will eventually need to handle the JWT refresh route which is going to be structured differently